### PR TITLE
DDCE-1491: GOVUK Assets upgrade

### PR DESCRIPTION
**Pull request by Ufuoma-Daniel**
- Doing the dependency updates was causing some issues that I didn't want to block this ticket on.
- I also deleted some unused filters and swapped to just bringing in 'frontendfilters' rather than each filter individually in application.conf.

### [Journey test changes to be merged first](https://github.com/hmrc/request-corporation-tax-number-journey-tests/pull/21)

 - [x]  I've made every effort to commit high quality, clean code and I have executed relevant static analyses to be sure
 - [x]  I've included appropriate tests with any code I've added (Unit, Integration, Acceptance etc.)
 - [x]  I've executed the acceptance test pack locally to ensure there are no functional regressions (Performance tests where relevant)
 - [x]  I've squashed my commits - including the JIRA issue number in the commit message
 - [ ]  I've run a dependency check to ensure all dependencies are up to date

### CODE REVIEW

File: conf/app.routes [hunk 1]
```diff
@@ -1,4 +1,6 @@
 # microservice specific routes
+->         /govuk-frontend                              govuk.Routes
```
Comment (DannyPennington97) [inline conf/app.routes:2]: Do these two not go in the prod.routes?

**Comment by DannyPennington97** [comment]
Screenshot 2021-05-04 at 09 58 57
The confirmation page is also looking weird for me, is it ok on your end?

File: conf/app.routes (hunk 1 above)
Comment (Ufuoma-Daniel) [reply conf/app.routes:2]: I was following this README and it said to put it there: https://github.com/hmrc/play-frontend-hmrc/blob/master/README.md

File: test/filters/SessionIdFilterSpec.scala [hunk 2]
```diff
@@ -39,10 +39,10 @@ object SessionIdFilterSpec {
 
   val sessionId = "28836767-a008-46be-ac18-695ab140e705"
 
-  class Filters @Inject()(sessionId: SessionIdFilter) extends DefaultHttpFilters(sessionId)
```
Comment (DannyPennington97) [inline test/filters/SessionIdFilterSpec.scala:42]: I feel these changes are left over from your filter testing?

File: .bsp/sbt.json [hunk 3]
```diff
@@ -0,0 +1 @@
+{"name":"sbt","version":"1.4.9","bspVersion":"2.0.0-M5","languages":["scala"],"argv":["/Library/Java/JavaVirtualMachines/jdk1.8.0_202.jdk/Contents/Home/jre/bin/java","-Xms100m","-Xmx100m","-classpath","/Users/udaniel/Library/Application Support/JetBrains/IntelliJIdea2020.3/plugins/Scala/launcher/sbt-launch.jar","xsbt.boot.Boot","-bsp"]}
```
Comment (DannyPennington97) [inline .bsp/sbt.json:1]: I think this is one we need to add to the gitignore and not have here, it has user specific paths in

File: app/views/ConfirmationView.scala.html [hunk 4]
```diff
@@ -17,34 +17,32 @@
...
-    <h2 class="heading-medium">@messages("confirmation.whatHappensNext.title")</h2>
+    <h2 class="govuk-heading-m">@messages("confirmation.whatHappensNext.title")</h2>
 
-    <p>@messages("confirmation.whatHappensNext.line1")</p>
+    <p class="govuk-body">@messages("confirmation.whatHappensNext.line1")</p>
 
     <div>
-        <p>@messages("confirmation.whatHappensNext.line2")
+        <p class="govuk-body">@messages("confirmation.whatHappensNext.line2")
             <a target="_blank" id="companies-house-search" href="@appConfig.companyHouseRegisterUrl">
```
Comment (DannyPennington97) [inline app/views/ConfirmationView.scala.html:39]: This <a> needs the `govuk-link` class adding I think, currently it's the wrong blue

File: .bsp/sbt.json (hunk 3 above)
Comment (Ufuoma-Daniel) [reply .bsp/sbt.json:1]: I didn't even notice a new file was generated, good catch. I'll sort it.

**Review by DannyPennington97 — APPROVED** [review]
Good to go, awaiting design team sign off before merging

**Pull request merged**